docker/instant-seal-node.dockerfile

```diff
@@ -7,11 +7,15 @@ FROM --platform=linux/amd64 ubuntu:22.04 AS base
 LABEL maintainer="Frequency"
 LABEL description="Frequency collator node in instant seal mode"
 
-RUN apt-get update && apt-get install -y ca-certificates && update-ca-certificates
+RUN apt-get update && apt-get install -y ca-certificates jq curl && update-ca-certificates
 
 # This is the 2nd stage: a very small image where we copy the Frequency binary
 FROM --platform=linux/amd64 ubuntu:22.04
 
+# We want jq and curl in the final image, but we don't need the support files
+RUN apt-get update && apt-get install -y jq curl && apt-get clean
+RUN rm -rf /usr/share/doc /usr/share/man /usr/share/zsh
```

Comment:
These would need to be one line in order not to make a layer without those additional files.

I think without it being one line, the image size will not change

Reply:
Fixed.